lib/open_project/plugins/acts_as_op_engine.rb

```diff
@@ -67,9 +93,9 @@ def self.included(base)
         end
       end
 
-      base.send(:define_method, :patch_with_namespace) do |*args|
+      def patch_with_namespace(*args)
```

Comment:
Assignment Branch Condition size for patch_with_namespace is too high. [15.84/15]